### ISSUE TITLE
Play all on Songlist/Genre/Artist

### ISSUE THREAD

**Issue by e-anima** [ampache/ampache#1432]
Is it possible to "play all" songs on the Songlist/Genre/Artist pages?
This is possible on playlists, but for example if i want to play all Metal Albums from all artist... that is not possible?

**Comment by Phyks** [comment]
I don't think so. I guess main reason is that Ampache playback is built on top of playlists and "play all" on these pages would potentially mean creating a huge temporary playlist.
